src/app/modules/ix-forms/components/ix-input-group/ix-input-group.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'ix-input-group',
+  templateUrl: './ix-input-group.component.html',
+  styleUrls: ['./ix-input-group.component.scss'],
+})
+export class IxInputGroupComponent {
```

Comment:
I dunno about `ix-input-group`. Feels a bit specific for a core package. How about extracting ix-label instead and doing grouping in snapshot-form component itself?

Reply:
Removed `ix-input-group` and grouped simply.